buildscripts/incremental/test.cmd

```diff
@@ -17,8 +17,10 @@ set PYTHONFAULTHANDLER=1
 @rem enable new style error handling
 set NUMBA_CAPTURED_ERRORS=new_style
 
-@rem First check that the test discovery works
-python -m numba.tests.test_runtests
+@rem First run Numba's Power-On-Self-Test to make sure testing will likely work
```

Comment:
Is this file used?

Reply:
The `test.cmd` doesn’t seem to be referenced anywhere. But, i’ll ignore it for the context of this PR and open a different PR to try to remove it.